custom/conf/app.example.ini

```diff
@@ -412,6 +412,10 @@ USER = root
 ;;
 ;; Whether execute database models migrations automatically
 ;AUTO_MIGRATION = true
+;;
+;; Threshold value (in seconds) beyond which query execution time is logged as a warning in the xorm logger
+;;
+;SLOW_QUERY_TRESHOLD = 5s
```

Comment:
Well, I think it is a typo ..... and it needs to be fixed at least before merging.

`THRESHOLD`